lib/pod/command/dependencies.rb

```diff
@@ -71,7 +75,8 @@ def run
         end
         graphviz_image_output if @produce_image_output
         graphviz_dot_output if @produce_graphviz_output
-        yaml_output
+        yaml_output if @list
```

Comment:
this is a breaking change -- we should default to list if summary isn't given

Reply:
Ah right, fixed it